File: lessons/compiled_lang/cs/src/m10.function.cs
using System;

class M10Function
{
    static string Capitalize(string s)
    {
        return s.ToUpper();
    }

    static void Main()
    {
        string s = "ibm";
        Console.WriteLine("The current string is: \"" + s + "\".");

        string result = Capitalize(s);
        Console.WriteLine("The capitalized string is: \"" + result + "\".");
    }
}

File: lessons/compiled_lang/java/src/m10.function.java
// Not public - see a00.output.java for why.
class M10Function {
    static String capitalize(String s) {
        return s.toUpperCase();
    }

    public static void main(String[] args) {
        String s = "ibm";
        System.out.println("The current string is: \"" + s + "\".");

        String result = capitalize(s);
        System.out.println("The capitalized string is: \"" + result + "\".");
    }
}
